src/GLX/libglxmapping.c

```diff
@@ -447,6 +447,37 @@ static GLboolean LookupVendorEntrypoints(__GLXvendorInfo *vendor)
     return GL_TRUE;
 }
 
+static Bool CheckVendorImportsTable(__GLXvendorNameHash *pEntry)
+{
+    // Make sure all the required functions are there.
+    if (pEntry->imports.isScreenSupported == NULL
+            || pEntry->imports.getProcAddress == NULL
+            || pEntry->imports.getDispatchAddress == NULL
+            || pEntry->imports.setDispatchIndex == NULL)
+    {
```

Comment:
Nitpick: I'd move the opening curly bracket to the previous line - ... == NULL) {
It seems to be the style used throughout the project.